google/cloud/spanner/database_admin_client.h

```diff
@@ -61,8 +61,8 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
  * This class uses `StatusOr<T>` to report errors. When an operation fails to
```

Comment:
Note to self:  Perhaps we should trying to suppress the generated documentation for this deprecated class, or at least have the documentation so marked.

Reply:
Done.  Maybe you can help with #10755 instead?

Reply:
> Maybe you can help with https://github.com/googleapis/google-cloud-cpp/issues/10755 instead?

Ack.